packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/assetModelDataStreamExplorer/assetModelPropertiesExplorer/assetModelPropertiesTable/assetModelPropertiesTable.tsx

```diff
@@ -112,8 +111,8 @@ export function AssetModelPropertiesTable({
         isModeledPropertyInvalid(item.dataType, selectedWidgets?.at(0)?.type)
       }
       onSelectionChange={(event) => {
-        const selectedAssets = event.detail.selectedItems;
-        onSelectAssetModelProperties(selectedAssets);
+        // const selectedAssets = event.detail.selectedItems;
```

Comment:
nit: remove comments

Reply:
This file needs to be deleted. Old asset model properties table is replaced.